src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.DesignTime.targets

```diff
@@ -21,6 +21,12 @@ Copyright (c) .NET Foundation. All rights reserved.
       we use the lack of the capability to detect downlevel scenarios.
     -->
     <ProjectCapability Include="DotNetCoreRazor"/>
+
+    <!--
+      Defines the ability to understand the configuration for the Razor language service provided by
+      the runtime/toolset packages. Introduced in 2.1
+    -->
+    <ProjectCapability Include="DotNetCoreRazorConfiguration"/>
```

Comment:
Should this be conditional? How would this work for a project targeting 2.0 but using a new VS/new SDK?

Reply:
Yup. I can move this to `.Configuration.targets`